schema.graphql

```diff
@@ -162,6 +162,10 @@ type Trade @entity {
   tradeBatchId: BigInt!
   tradeEpoch: BigInt!
 
+  # Tokens
+  buyToken: Token!
+  sellToken: Token!
```

Comment:
This is derived from the order. They made it just easier to get from the events, but we already had that information (through the order)

Reply:
Agree,done